mon-pix/app/components/module/stepper.gjs

```diff
@@ -24,11 +24,38 @@ export default class ModulixStepper extends Component {
     return this.stepsToDisplay.length < this.args.steps.length;
   }
 
+  get answerableElementsInCurrentStep() {
+    const currentStep = this.stepsToDisplay[this.stepsToDisplay.length - 1];
+    return currentStep.elements.filter((element) => element.isAnswerable);
+  }
+
+  get hasAnswerableElementInCurrentStep() {
+    return this.answerableElementsInCurrentStep.length > 0;
+  }
+
+  get allAnswerableElementsAreAnsweredInCurrentStep() {
+    return this.answerableElementsInCurrentStep.some((element) => {
+      return this.args.passage.getLastCorrectionForElement(element) !== undefined;
+    });
+  }
+
+  get shouldDisplayNextButton() {
+    return (
+      this.hasNextStep &&
+      (!this.hasAnswerableElementInCurrentStep || this.allAnswerableElementsAreAnsweredInCurrentStep)
```

Comment:
`!this.hasAnswerableElementInCurrentStep` ne sert à rien, si ? :)

Reply:
Pourquoi selon toi ?

Reply:
Je n'avais pas prévu tant de questions :p